### PR TITLE
pgvectorscale client added

### PR DESCRIPTION
**Pull request by Sheharyar570**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ All the database client supported
 |elastic|`pip install vectordb-bench[elastic]`|
 |pgvector|`pip install vectordb-bench[pgvector]`|
 |pgvecto.rs|`pip install vectordb-bench[pgvecto_rs]`|
+|pgvectorscale|`pip install vectordb-bench[pgvectorscale]`|
```
Comment (wahajali) [inline README.md]: @Sheharyar570 is this required?

File: vectordb_bench/backend/clients/pgvectorscale/config.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            },
+        }
+    
+    def search_param(self) -> dict:
+        return {
+            "metric": self.parse_metric(),
+            "metric_fun_op": self.parse_metric_fun_op(),
+        }
+    
+    def session_param(self) -> dict:
```
Comment (wahajali) [inline vectordb_bench/backend/clients/pgvectorscale/config.py:103]: @Sheharyar570 haven't seen `session_param` before. Is this defined for other clients as well?

File: README.md (hunk 1 above)
Comment (Sheharyar570) [reply README.md]: Not required, but an instruction if anyone wants to install only pgvectorscale client.

File: vectordb_bench/backend/clients/pgvectorscale/config.py (hunk 2 above)
Comment (Sheharyar570) [reply vectordb_bench/backend/clients/pgvectorscale/config.py:103]: Yes it is, defined in pgvector client.

**Pull request merged**